mappings/dataprepper/README.md

```diff
@@ -0,0 +1,104 @@
+- **Load**: The transformed data is loaded into sinks such as Elasticsearch, Kafka, or files for storage, analysis, or further processing.
+
+## Processors
+
+Processors perform an action on your data, such as filtering, transforming, or enriching.
+
+### Append Processor
+    processors:
+      - append:
+          field: "new_field"
+          value: "example_value"
+
+### Convert Processor
+    processors:
+      - convert:
+          field: "field_name"
+          type: "integer"
+
+### Date Processor
+    processors:
+      - date:
+          field: "date_field"
+          target_field: "new_date_field"
+          formats:
+            - "ISO8601"
+
+### Grok Processor
+    processors:
+      - grok:
+          field: "message"
+          patterns:
+            - "%{IP:client_ip} %{USER:client_user}"
+
+### Remove Processor
+    processors:
+      - remove:
+          field: "field_name"
+
+### Rename Processor
+    processors:
+      - rename:
+          field: "old_field"
+          target_field: "new_field"
+
+
+## Inputs
+
+Data Prepper supports various input sources for ingesting data into the pipeline. Some of the supported input types are:
+
+### 1. Kafka Input
+
+You can use the Apache Kafka source (kafka) in Data Prepper to read records from one or more Kafka topics. These records hold events that your Data Prepper pipeline can ingest. The kafka source uses Kafka’s Consumer API to consume messages from the Kafka broker, which then creates Data Prepper events for further processing by the Data Prepper pipeline.
+
+### 2. HTTP Input
+
+http_source is a source plugin that supports HTTP. Currently, http_source only supports the JSON UTF-8 codec for incoming requests, such as [{"key1": "value1"}, {"key2": "value2"}]. The following table describes options you can use to configure the http_source source.
+
+## Sinks
+
+Data Prepper supports multiple sink types for storing or forwarding processed data. Some of the available sink options include:
+
+### 1. Opensearch Sink
+
+You can use the opensearch sink plugin to send data to an OpenSearch cluster, a legacy Elasticsearch cluster, or an Amazon OpenSearch Service domain.
+
+### 2. S3 Sink
+
+The s3 sink saves and writes batches of Data Prepper events to Amazon Simple Storage Service (Amazon S3) objects. The configured codec determines how the s3 sink serializes the data into Amazon S3.
+
+### 3. File Sink
+
+Use the file sink to create a flat file output, usually a .log file.
+
+## Getting Started
+
+To get started with Data Prepper, follow these steps:
+
+1. Clone the Data Prepper repository.
+2. Install any necessary dependencies.
+3. Configure your inputs, processors, and sinks in the Data Prepper configuration file.
+4. Run Data Prepper with your configuration to start ingesting and processing data.
+
+For detailed installation instructions and usage guidelines, refer to the official documentation.
+
+## Contributing
```

Comment:
These appear to be from the dataprepper repo's readme and are not valid for this repo. 
Let's please remove non relevant sections from this read me file. @adplotzk